### ISSUE TITLE
Does not work in Swift

### ISSUE THREAD

**Issue by vandyshev** [venmo/VENTouchLock#42]
```
let viewController = VENTouchLockCreatePasscodeViewController()
    if let navigationController = self.navigationController {
navigationController.pushViewController(viewController, animated: true)
}
```

```
*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Could not load NIB in bundle: 'NSBundle </Users/evgenii/Library/Developer/CoreSimulator/Devices/F0D6421C-E864-4818-A22F-137647C02EA4/data/Containers/Bundle/Application/D7448CC5-BC1B-43DC-80DA-38E8E809ECB7/Roscredit.app> (loaded)' with name 'VENTouchLockPasscodeView''
*** First throw call stack:
(
	0   CoreFoundation                      0x00000001033b3a75 __exceptionPreprocess + 165
	1   libobjc.A.dylib                     0x0000000105179bb7 objc_exception_throw + 45
	2   CoreFoundation                      0x00000001033b39ad +[NSException raise:format:] + 205
	3   UIKit                               0x000000010417b133 -[UINib instantiateWithOwner:options:] + 552
	4   UIKit                               0x000000010417cf92 -[NSBundle(UINSBundleAdditions) loadNibNamed:owner:options:] + 160
	5   VENTouchLock                        0x0000000102e32ba1 -[VENTouchLockPasscodeView initWithTitle:frame:titleColor:characterColor:] + 289
	6   VENTouchLock                        0x0000000102e3327f -[VENTouchLockPasscodeView initWithTitle:frame:] + 207
	7   VENTouchLock                        0x0000000102e33353 -[VENTouchLockPasscodeView initWithFrame:] + 99
	8   VENTouchLock                        0x0000000102e333ec -[VENTouchLockPasscodeView init] + 92
	9   VENTouchLock                        0x0000000102e349c0 -[VENTouchLockPasscodeViewController configurePasscodeView] + 64
	10  VENTouchLock                        0x0000000102e34456 -[VENTouchLockPasscodeViewController viewDidLoad] + 406
	11  VENTouchLock                        0x0000000102e2f1c6 -[VENTouchLockCreatePasscodeViewController viewDidLoad] + 54
	12  UIKit                               0x0000000103fda580 -[UIViewController loadViewIfRequired] + 738
```

**Comment by vandyshev** [comment]
If you don't use the *use_frameworks!* in **Podfile**, it works well

**Comment by Jasdev** [comment]
@vandyshev Be sure to add `import VENTouchLock` to your appropriate Swift files (in addition to import statement in Bridging Header). This did the trick for me :smile:

**Comment by vandyshev** [comment]
I'm sure that the added import. And in Bridging Header too. The problem in that app can't get Nib from framework

**Comment by dasmer** [comment]
Fixed in #43
